### DISCUSSION TITLE
Visualize the loss curves

### DISCUSSION THREAD

**Discussion by ShayAdaddi** [sintel-dev/Orion#542] [Q&A] [answered]
Hi,

Is there a simple method to grab the loss values and plot the curves ? 
I'm using 'aer' and I see the loss prints to the console so I'm sure this info is located somwhere.

Thanks !

**Comment by sarahmish** [comment, marked as answer]
Hi @ShayAdaddi –

The straightforward approach is to get a hold of the `keras.History` object. To do that, select the AER block directly and reference the `fit_history`, below is is the loss from the readme example:

```python3
In [3]: orion._mlpipeline.blocks['orion.primitives.aer.AER#1'].instance.fit_history.history
Out[3]: 
{'loss': [0.46820536255836487,
  0.4265056252479553,
  0.41729703545570374,
  0.410270094871521,
  0.4081384539604187],
 'tf.__operators__.getitem_loss': [0.47297951579093933,
  0.4305891692638397,
  0.41900143027305603,
  0.40642619132995605,
  0.4008139371871948],
 'tf.__operators__.getitem_1_loss': [0.4683806598186493,
  0.4258121848106384,
  0.4177078306674957,
  0.416854590177536,
  0.4176575839519501],
 'tf.__operators__.getitem_2_loss': [0.46308082342147827,
  0.4238089919090271,
  0.4147711396217346,
  0.40094512701034546,
  0.3964250385761261],
 'val_loss': [0.4367644786834717,
  0.4254518151283264,
  0.4154045581817627,
  0.41593384742736816,
  0.4100659191608429],
 'val_tf.__operators__.getitem_loss': [0.42555269598960876,
  0.424095094203949,
  0.405383825302124,
  0.39520588517189026,
  0.39268115162849426],
 'val_tf.__operators__.getitem_1_loss': [0.44126948714256287,
  0.4218960404396057,
  0.41904327273368835,
  0.429433673620224,
  0.420706570148468],
 'val_tf.__operators__.getitem_2_loss': [0.4389663636684418,
  0.43392014503479004,
  0.418148010969162,
  0.40966203808784485,
  0.40616926550865173]}
```

**Reply by ShayAdaddi** [reply]
Thank you it works great !
